README.md

```diff
@@ -21,17 +21,39 @@ $ npm install eslint-plugin-react --save-dev
 
 # Configuration
 
-Add `plugins` section and specify ESLint-plugin-React as a plugin.
+Enable JSX support.
+
+With ESLint 2+
 
 ```json
 {
-  "plugins": [
```

Comment:
this is definitely necessary, and will not work without it being present somewhere (note that if using a shared config, like the airbnb config, it might be declaring this for you)

Reply:
It's not necessary when using the recommended rule config (it defines itself). I'll edit to make that clearer.

Reply:
gotcha, thanks.